Makefile

```diff
@@ -10,7 +10,7 @@ CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 .PHONY: dsp-stack
 dsp-stack: ## clone the dsp-api git repository and run the dsp-stack
 	@mkdir -p .tmp
-	@git clone --branch main --single-branch --depth 1 https://github.com/dasch-swiss/dsp-api.git .tmp/dsp-stack
+	@git clone --branch v24.0.8 --single-branch https://github.com/dasch-swiss/dsp-api.git .tmp/dsp-stack
```

Comment:
This is temporary, I plan to revert this as soon as the other PR is merged that fixes the project IRIs. But as long as the API is not released, I don't want to merge the other PR, so that I could also make a release of dsp-tools before the API is released.